pkg/comp-functions/functions/vshnkeycloak/deploy.go

```diff
@@ -235,43 +351,21 @@ func newValues(ctx context.Context, svc *runtime.ServiceRuntime, comp *vshnv1.VS
 			},
 		},
 		"nodeSelector": nodeSelector,
-		"metrics": map[string]any{
+		"dbchecker": map[string]any{
 			"enabled": true,
-			"serviceMonitor": map[string]any{
-				"enabled": true,
-			},
-		},
-		"httpRelativePath": comp.Spec.Parameters.Service.RelativePath,
-		"postgresql": map[string]any{
-			"enabled": false,
 		},
-		"externalDatabase": map[string]any{
-			"host":     string(cd[vshnpostgres.PostgresqlHost]),
-			"port":     string(cd[vshnpostgres.PostgresqlPort]),
-			"database": string(cd[vshnpostgres.PostgresqlDb]),
-			"user":     string(cd[vshnpostgres.PostgresqlUser]),
-			"password": string(cd[vshnpostgres.PostgresqlPassword]),
-		},
-		"extraVolumeMounts": []map[string]string{
-			{
-				"name":      "postgresql-certs",
-				"mountPath": "/opt/bitnami/keycloak/certs/",
-			},
-		},
-		"extraVolumes": []map[string]any{
-			{
-				"name": "postgresql-certs",
-				"secret": map[string]any{
-					"secretName":  pgSecretName,
-					"defaultMode": 420,
+		// See https://github.com/keycloak/keycloak/issues/11286
+		// readOnlyRootFilesystem: true
```

Comment:
Why is this field even there and commented out?

Reply:
I copy pasted it over from Inventages values.

Might be something to change once the bug is resolved.

Reply:
Ok. Fine by me